### PR TITLE
Install ngrx/store into our app #108

### PR DESCRIPTION
**Pull request by girum11**
Changed CanvasService state from Card[] to ngrx/store Action/Reducer.

### CODE REVIEW

File: src/client/app/graph/canvas/canvas.component.html [hunk 1]
```diff
@@ -4,5 +4,5 @@
      xmlns:xlink="http://www.w3.org/1999/xlink"
      (mousedown)="this.onMousedown($event)"
      (mousewheel)="this.onMousewheel($event)">
-  <g uxg-card *ngFor="let card of this.canvasService.cards" [card]="card"></g>
+  <g uxg-card *ngFor="let card of this.canvasService.cards | async" [card]="card"></g>
```
Comment (nati22) [inline src/client/app/graph/canvas/canvas.component.html]: What's the "| async" doing?

File: src/client/app/app.module.ts [hunk 2]
```diff
@@ -17,7 +19,10 @@ import {SharedModule} from './shared/shared.module';
     RouterModule.forRoot(routes),
     GraphModule,
     HomeModule,
-    SharedModule.forRoot()
+    SharedModule.forRoot(),
+    StoreModule.provideStore({
+      'cards': cardsReducer
```
Comment (eyuelt) [inline src/client/app/app.module.ts]: Use a constant for 'cards'.

File: src/client/app/graph/canvas/canvas.component.html (hunk 1 above)
Comment (girum11) [inline src/client/app/graph/canvas/canvas.component.html]: We chatted about this in person. "async" is a pipe that automatically "unwraps" this Observable:

https://angular.io/docs/ts/latest/api/common/index/AsyncPipe-pipe.html

File: src/client/app/app.module.ts (hunk 2 above)
Comment (girum11) [inline src/client/app/app.module.ts]: Done. We'll have to used computed properties for this.

**Pull request merged**